.gitignore

```diff
@@ -13,3 +13,9 @@ yarn-debug.log*
 yarn-error.log*
 store.db
 store.db.compacted
+_book
+docs/html
```

Comment:
I believe this will create an issue - it will build on CI but not add the HTML :( It simply does a git add, since these are ignored, they will not be added and pushed.